spec.bs

```diff
@@ -2357,3 +2372,88 @@ achieve the outcomes described in the above explanatory content.
   /fenced-frame/permission-geolocation.https.html
   /fenced-frame/permission-notification.https.html
 </wpt>
+
+<h3 id=private-network-access-monkeypatch>Private Network Access</h3>
+
+[[Private-Network-Access]] is a security feature that regulates when and how [=local network
+requests=] can be made. On [[HTML#creating-browsing-contexts|creation]], <{iframe}>s inherit their
+[=Document/policy container=] from their [=navigable container=]'s [=node document=], while
+<{fencedframe}>s do not. While this prevents information from leaking into a <{fencedframe}> from
+its embedder, it also results in a [=Document/policy container=] for {{Document}}s inside of a
+<{fencedframe}> being created that causes [=subresource requests=] to not do private network access
+checks. This section patches the [[Private-Network-Access]] specification to make subresources be
```

Comment:
From https://docs.google.com/document/d/1Fy9CC5w1-8C1s2YfRioeURUVN5I3u3jA5o_87kH9-fA/edit#heading=h.uirj6a11h4rr:

"This currently results in the use of a default PolicyContainer, for which the IP address space is `kUnknown`. Due to the existing carveout for `kUnknown`, this means fenced frames’ subresource fetches are not subject to PNA checks."

Reply:
Sure but that sounds like an implementation detail that doesn't exist in the specs, since `kUnknown` is not a property in the spec and the default policy container value is always public: https://wicg.github.io/local-network-access/#policy-container-ip-address-space

Reply:
Resolving because this is an outdated comment.